npc/cities/morocc.txt

```diff
@@ -131,7 +131,7 @@ morocc,223,102,0	script	Volunteer - Morocc#06::MocVolunteer	741,{
 	specialeffect EF_HIT2,AREA,"Volunteer - Morocc#06";
 	next;
 	mes "[Morocc Volunteer]";
-	mes "Heyyy, you're being too harsh, Senyorita. Your fist may be as fragile as you are, but it sure can hurt somebody.";
+	mes "Heyyy, you're being too harsh, Señorita. Your fist may be as fragile as you are, but it sure can hurt somebody.";
```

Comment:
For that to show correctly you probably have to change the enconding to ANSI. Not sure if will work for all langtypes tho.

Reply:
Could you try at least langtype 1?